pom.xml

```diff
@@ -25,7 +25,7 @@ SOFTWARE.
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <properties>
-    <rxjava.version>3.0.0-RC6</rxjava.version>
+    <vertx.version>4.0.0-SNAPSHOT</vertx.version>
```

Comment:
@Sammers21 is it a really good decision to use snapshot version of `vertx`? Why we need exactly snapshot version?

Reply:
This code :
```
asyncFile.toSubscriber()                            .onWriteStreamEnd(emitter::onComplete)                          .onWriteStreamError(emitter::onError)
```
Won't compile without it

Reply:
@Sammers21 got it, thanks.